src/main/java/ru/sbtqa/tag/pagefactory/stepdefs/GenericStepDefs.java

```diff
@@ -0,0 +1,210 @@
+    }
+
+    @И("^(?:пользователь |он |)в списке \"([^\"]*)\" находит элемент со значением \"([^\"]*)\"$")
+    @And("^user in list \"([^\"]*)\" finds the value element \"([^\"]*)\"$")
+    public void findElementInList(String listTitle, String value) throws PageException {
+        boolean found = false;
+        for (WebElement el : PageFactory.getInstance().getCurrentPage().findListOfElements(listTitle)) {
+            if (el.getText().equals(value)) {
+                found = true;
+                break;
+            }
+        }
+        if (!found) {
+            throw new AutotestError(String.format("Element with text '%s' is absent in list '%s'", value, listTitle));
+        }
+    }
+
+    @И("^(?:пользователь |он |)(?:находится на странице|открывается страница|открывается вкладка мастера) \"(.*?)\"$")
+    @And("^(?:user |he |)(?:is on the page|page is being opened|master tab is being opened) \"(.*?)\"$")
+    public void openPage(String title) throws PageInitializationException {
+        if (!PageFactory.getWebDriver().getWindowHandles().isEmpty()) {
+            PageFactory.getWebDriver().getWindowHandles().stream()
+                    .forEach(PageFactory.getWebDriver().switchTo()::window);
+        }
+        PageFactory.getInstance().getPage(title);
+    }
+
+    @И("^(?:пользователь |он |)\\((.*?)\\)$")
+    @And("^user \\((.*?)\\)$")
+    public void userActionNoParams(String action) throws PageInitializationException, NoSuchMethodException {
+        PageFactory.getInstance().getCurrentPage().executeMethodByTitle(action);
+    }
+
+    @И("^(?:пользователь |он |)\\((.*?)\\) (?:с параметром |)\"([^\"]*)\"$")
+    @And("^user \\((.*?)\\) (?:with param |)\"([^\"]*)\"$")
+    public void userActionOneParam(String action, String param) throws PageInitializationException, NoSuchMethodException {
+        PageFactory.getInstance().getCurrentPage().executeMethodByTitle(action, param);
+    }
+
+    @И("^(?:пользователь |он |)\\((.*?)\\) (?:с параметрарми |)\"([^\"]*)\" \"([^\"]*)\"$")
+    @And("^user \\((.*?)\\) (?:with the parameters |)\"([^\"]*)\" \"([^\"]*)\"$")
+    public void userActionTwoParams(String action, String param1, String param2) throws PageInitializationException, NoSuchMethodException {
+        PageFactory.getInstance().getCurrentPage().executeMethodByTitle(action, param1, param2);
+    }
+
+    @И("^(?:пользователь |он |)\\((.*?)\\) (?:с параметрарми |)\"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\"$")
+    @And("^user \\((.*?)\\) (?:with the parameters |)\"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\"$")
+    public void userActionThreeParams(String action, String param1, String param2, String param3) throws PageInitializationException, NoSuchMethodException {
+        PageFactory.getInstance().getCurrentPage().executeMethodByTitle(action, param1, param2, param3);
+    }
+
+    @И("^(?:пользователь |он |)\\((.*?)\\) данными$")
+    @And("^user \\((.*?)\\) data$")

```

Comment:
It will not work for steps without the user (пользователь/он)

^(?:пользователь |он |)\((.?)\) данными$
^(?:user |) \((.?)\) data$

here we lose steps without user
^(?:пользователь |он)\((.?)\) данными$
^user \((.?)\) data$

the explanation can be found @iamanikeev